src/game/utils/nat.c

```diff
@@ -121,9 +121,12 @@ int readpmpresponse(nat_ctx *ctx, natpmpresp_t *response) {
 #else
         FD_SET(ctx->natpmp.s, &fds);
 #endif
-        getnatpmprequesttimeout(&ctx->natpmp, &timeout);
-        select(FD_SETSIZE, &fds, NULL, NULL, &timeout);
-        r = readnatpmpresponseorretry(&ctx->natpmp, response);
+        // getnatpmprequesttimeout(&ctx->natpmp, &timeout);
+        timeout.tv_sec = 3;
```

Comment:
do we even need 3 seconds ?

Reply:
yeah, probably, although some routers are *slow*